proposals/text/5289-recency-reranker.md

```diff
@@ -0,0 +1,57 @@
+- Title: Addition of a 'RecencyReranker' node
+- Decision driver: @elundaeva
+- Start Date: 2023-07-05
+- Proposal PR: https://github.com/deepset-ai/haystack/pull/5289
+- Github Issue or Discussion: some links available in the "Detailed design" section below
+
+# Summary
+
+This reranker allows to have retrieved documents sorted not only by relevance (default) but also with recency factored in.
+
+# Basic example
+
+The RecentnessReranker would be instantiated as follows:
```

Comment:
I think it would be great to also show an example of a full pipeline would look with this node in it.